modules/appeals_api/config/schemas/v2/10182.json

```diff
@@ -80,19 +80,19 @@
-                    },
-                    "representative": {
-                      "type": "object",
-                      "additionalProperties": false,
-                      "properties": {
-                        "name": { "type": "string", "maxLength": 120 }
-                      }
                     }
                   },
                   "required": ["homeless",  "phone", "email"],
                   "if": { "properties": { "homeless": { "const": false } } },
                   "then": { "required": ["address"] }
                 },
+                 "representative": {
```

Comment:
This had been left within `veteran`. Probably not caught since it's not being tested?